.github/workflows/continuous-integration.yml

```diff
@@ -16,24 +16,35 @@ jobs:
 
     steps:
       - name: "Checkout"
-        uses: "actions/checkout@v2.0.0"
+        uses: "actions/checkout@v2"
 
       - name: "Install PHP with extensions"
-        uses: "shivammathur/setup-php@1.8.1"
+        uses: "shivammathur/setup-php@v1"
         with:
           coverage: "none"
           extensions: "simplexml"
           php-version: "7.0"
 
-      - name: "Cache dependencies installed with composer"
```

Comment:
deleted this unnecessary complexity, because the build is wicked fast without caching